tests/test_farmer.py

```
@@ -0,0 +1,172 @@
+        await db_session.commit()
+        await db_session.refresh(farmer)
+
+        # Refresh user to get the relationship
+        await db_session.refresh(user)
+        
+        # Verify relationship - use explicit query to avoid lazy loading issues
+        from sqlalchemy import select
+        user_with_farmer = await db_session.execute(
+            select(User).where(User.id == user.id)
+        )
+        user_obj = user_with_farmer.scalar_one()
```

Ruff (0.12.2)

Local variable `user_obj` is assigned to but never used

Remove assignment to unused variable `user_obj`

(F841)